src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+                  <span v-if="index === 0"> {{ word }}</span>
+                  <span v-else>, {{ word }}</span>
+                </b>
+              </v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="3">
+            <v-row justify="end" no-gutters>
+              <v-col v-if="item.count" cols="auto" class="pt-1">
+                <v-chip class="chip-count" :color="item.iconColor" dark>{{ item.count }}</v-chip>
+              </v-col>
+              <v-col v-if="!isExpanded && item.expandLabel" cols="auto">
+                <v-row no-gutters :justify="item.count ? 'center' : 'end'">
+                  <v-col class="expand-label" :class="item.count ? 'pl-4' : ''" cols="auto">
+                    {{ item.expandLabel.closed }}
+                  </v-col>
+                  <v-col class="pl-3 pr-5 pt-1" cols="auto">
+                    <v-icon class="expand-icon">mdi-chevron-down</v-icon>
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col v-else-if="item.expandLabel" cols="auto">
+                <v-row no-gutters :justify="item.count ? 'center' : 'end'">
+                  <v-col class="expand-label" :class="item.count ? 'pl-4' : ''" cols="auto">
+                    {{ item.expandLabel.open }}
+                  </v-col>
+                  <v-col class="pl-3 pr-5 pt-1" cols="auto">
+                    <v-icon class="expand-icon">mdi-chevron-up</v-icon>
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+          </v-col>
+        </v-row>
+      </tr>
+    </template>
+    <template v-slot:expanded-item="{ item }">
+      <v-row no-gutters class="pt-2">
+        <v-col class="pa-0">
+          <v-row v-if="item.expandedList" no-gutters class="px-15">
+            <QuickSearchNames :namesList="item.expandedList"/>
+          </v-row>
+          <v-row v-if="item.expandedInfo1"
+                 class="name-check-info-text pb-5 pl-13"
+                 :class="item.expandedList ? 'pt-7' : ''"
+                 no-gutters>
+            <v-col cols="auto">
+              <v-icon>mdi-information-outline</v-icon>
+            </v-col>
+            <v-col class="pl-2" style="max-width: 50rem">
+              <div v-if="item.info === ITEM_TYPES.SIMILAR_MATCH">
+                <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+                  <template v-slot:activator="{ on, attrs }">
+                    <span class="dotted-underline" v-bind="attrs" v-on="on">Similar names</span>
+                  </template>
+                  <p class="ma-0">
+                    Similar names contain the same, synonymous, or similar looking or sounding words.
+                  </p>
+                </v-tooltip>
+                in different business categories may be approved at the discretion of the
+                Business Registry. If you feel your name is unique within your business category,
```

Comment:
This sentence seems a little off, with the comma maybe?

Reply:
haha - it's copy/pasted from the UX copy so it's up to them